aries_vcx/src/utils/devsetup.rs

```diff
@@ -365,20 +369,27 @@ impl SetupProfile {
         cfg!(feature = "modular_libs_tests")
```

Comment:
this function will always return false now since we aren't using this flag anymore. 

This function is used by `init_holder_setup_in_indy_context` to decide we should run the holder POV using vdrtools or modular_libs. So always returning false means that a lot of tests with modular_libs are no longer being ran in CI, which is dangerous IMO.

I think changing this to `"modular_libs"` instead should work..

Reply:
Great catch again. It was a leftover. Fixed. Hopefully the CI will behave.

Reply:
Fixed.